charts/postgres-operator-ui/templates/deployment.yaml

```diff
@@ -19,6 +19,10 @@ spec:
       labels:
         app.kubernetes.io/name: {{ template "postgres-operator-ui.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
+      {{- with .Values.podAnnotations }}
```

Comment:
is it not {{- if .. ?

Reply:
Could be, but I personally prefer this syntax

https://helm.sh/docs/chart_template_guide/control_structures/#:~:text=Note%20that%20we%20removed%20the%20if%20conditional%20from%20the%20previous%20exercise%20because%20it%20is%20now%20unnecessary%20%2D%20the%20block%20after%20with%20only%20executes%20if%20the%20value%20of%20PIPELINE%20is%20not%20empty.